test/index.js

```diff
@@ -1,15 +1,14 @@
 'use strict';
 
-import test from 'tape';
+import test from 'blue-tape';
 import hello from '../source/index';
 
 
-test('Tests run', (assert) => {
-  assert.pass('Tests run');
-  assert.end();
-});
+test('Tests run', (assert) => Promise.resolve(
```

Comment:
Yes, allow `new` -- otherwise lots of stuff breaks, including all built-in constructors. );